validation/standard/standard-bytecodes.toml

```diff
@@ -1,3 +1,23 @@
+["op-contracts/v1.8.0"] # Holocene https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.8.0-rc.3
```

Comment:
I used this unadorned tag to stand in for this "line" of release candidates. It might make more sense to actually used the `-rc.3` suffix though.

Reply:
We decided to add the suffix back in, and we need to update Clabby's deployment script so it can pass in the tag with a suffix (right now it is hardcoded to `v1.8.0`).

Reply:
[2857b90](https://github.com/ethereum-optimism/superchain-registry/pull/704/commits/2857b904e145dca9bd5b3b1a80353898c99f5573)